PanModal/Presentable/PanModalPresentable+Defaults.swift

```diff
@@ -112,4 +112,7 @@ public extension PanModalPresentable where Self: UIViewController {
 
     }
 
+    func panModalDismissCompleted() {
```

Comment:
could we change this to `panModalDidDismiss` (for consistency)

Reply:
@ste57 
I rename `panModalDismissCompleted` `panModalDidDismiss`.
c0e8c09